zppy/templates/default.ini

```diff
@@ -23,6 +25,7 @@ input = string
 input_files = string(default="eam.h0")
 # The specific subdirectory with the atmospheric data
 input_subdir = string(default="archive/atm/hist")
+# machine -- NOTE: this parameter is created internally
```

Comment:
Just adding this comment for `machine`, `scriptDir`, and `templateDir` -- these parameters existed before this pull request. `bundle` is the only new parameter.